src/classes/button.py

```diff
@@ -0,0 +1,30 @@
+import pygame
+
+class Button:
+
+    def __init__(self, screen: pygame.Surface, color: tuple[int, int, int], hover_color: tuple[int, int, int], start_point: tuple[int, int], dim: tuple[int, int], text: str, on_click=None, cargs=()) -> None:
+        self.screen = screen
+        self.color = color
+        self.hover_color = hover_color
+        self.x = start_point[0]
+        self.y = start_point[1]
+        self.width = dim[0]
+        self.height = dim[1]
+        self.text = text
+        self.on_click = on_click
+        self.cargs = cargs
+
+    def is_selected(self) -> bool:
```

Comment:
use _is_selected for private function

Reply:
noted

Reply:
I think in this case we can change it to _is_selected since this function will only ever be called within the class

is_selected = public
_is_selected = private